src/roiextractors/extraction_tools.py

```diff
@@ -13,7 +14,10 @@
 except ImportError:
     HAVE_H5 = False
 try:
-    import scipy.io as spio
+    if hasattr(scipy.io.matlab, "mat_struct"):
```

Comment:
I, for one, very much appreciate that Python modules treat submodules as attributes.